python/pyspark/sql/connect/session.py

```diff
@@ -176,6 +180,27 @@ def enableHiveSupport(self) -> "SparkSession.Builder":
                 error_class="NOT_IMPLEMENTED", message_parameters={"feature": "enableHiveSupport"}
             )
 
+        def _apply_options(self, session: "SparkSession") -> None:
```

Comment:
BTW, we will have to fix `pyspark.sql.session.SparkSession.Builder.getOrCreate` too to pass the configurations properly.

Reply:
this help function is also used in `getOrCreate`